src/classes/PermanentFileSystem.ts

```diff
@@ -147,6 +147,40 @@ export class PermanentFileSystem {
     return populatedRecord;
   }
 
+  private async loadRecords(requestedPaths: string[]): Promise<Record[]> {
+    // See https://github.com/PermanentOrg/permanent-sdk/issues/73
+    // This implementation is intentionally sequential even though using
+    // requestePaths.map would allow us to run all of the lookups in parallel.
```

Comment:
(Tiny nitpick, not a blocker) Typo, should be `requestedPaths`

Reply:
Dang -- good eye, @cecilia-donnelly.